app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -235,11 +238,15 @@ private void animateGameItem(TwoStepAnimative object) {
 
     private void failGame() {
         isPlaying = false;
+        // One passed obstacle = 10 score points.
+        LevelManager.setCurrentLevelScore(score * 10);
```

Comment:
I am confused why we multiply here by 10 if the player has failed.

Reply:
We discussed with Max that the score ending with "0" looks better, so I multiply it by 10 everywhere. The current result is recorded for display on the screen. But only the best result stores in shared preferences.

Reply:
I see why don't we do score+=10 and we just do:        
LevelManager.setCurrentLevelScore(score);?

Reply:
Fixed